README.md

```diff
@@ -91,8 +98,24 @@ DATABASES['default']['NAME'] = 'var/workbench.db'
 ```
 
 Because this XBlock uses a Django model, you need to sync the database
-before starting the workbench. Run this from the XBlock repository
-root:
+before starting the workbench. `problem-builder` uses MySQL 5.6 and hence you can quickly spin up
+an instance of MySQL 5.6 with docker. Run the following command to start a MySQL 5.6 server.
+
+```bash
+docker run --rm -it -p 3306:3306 -e MYSQL_ROOT_PASSWORD=rootpw -e MYSQL_DATABASE=db circleci/mysql:5.6 mysqld --character-set-server=latin1 --collation-server=latin1_swedish_ci
```

Comment:
Is there a reason to not just use the standard mysql:5.6 image?
I think you can just as easily use:
```docker run --rm -it -p 3306:3306 -e MYSQL_ROOT_PASSWORD=rootpw -e MYSQL_DATABASE=db mysql:5.6```

Reply:
Additionally I would recommend not using the default port in case it's already used by an existing instance of mysql. So:

```suggestion
docker run --rm -it -p 3307:3306 -e MYSQL_ROOT_PASSWORD=rootpw -e MYSQL_DATABASE=db mysql:5.6
```

Reply:
@xitij2000 I have tested it with standard SQL image, and it works, thanks.
Updated the port too :+1: